docs/reference/ml/anomaly-detection/functions/ml-time-functions.asciidoc

```diff
@@ -77,6 +77,12 @@ its past behavior.
 The `time_of_week` function detects when events occur that are outside normal
 usage patterns. For example, it detects login events on the weekend.
 
+IMPORTANT: The `time_of_week` function is modeled in UTC and the duration of the 
+  week is in seconds. It means that the `typical` and `actual` values are 
+  seconds after a whole number of weeks after 1/1/1970 in UTC which is a 
+  Thursday. For example, a value of `475` is 475 seconds after midnight on 
+  Thursday in UTC. 
```

Comment:
The first sentence doesn't read quite right to me. How about:
```suggestion
IMPORTANT: The `time_of_week` function models time in epoch seconds modulo the duration
  of a week in seconds. It means that the `typical` and `actual` values are 
  seconds after a whole number of weeks since 1/1/1970 in UTC which is a 
  Thursday. For example, a value of `475` is 475 seconds after midnight on 
  Thursday in UTC. 
```